lib/mbta.js

```diff
@@ -3,6 +3,47 @@
 // Code here.
 
 const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+  const mbta =
+  {
+  "Red": ["South Station", "Park Street", "Kendall", "Central", "Harvard",
+  "Porter", "Davis", "Alewife"],
+
+  "Green": ["Government Center", "Park Street","Boylston", "Arlington",
+  "Copley", "Hynes", "Kenmore"],
+
+  "Orange": ["North Station", "Haymarket", "Park Street", "State",
+  "Downtown Crossing","Chinatown", "Back Bay", "Forest Hills"]
+  }
+  const intersection = 'Park Street'
+  // let startLine = 'redLine'
+  //let startStation = 'South Station'
+  // let endLine = 'redLine'
+  // let endStation = 'Park Street'
+  let startIntersectionIndex
+  let endIntersectionIndex
+  let startStationIndex
+  let endStationIndex
+
+  let findIndex = function(trainLine,station) {
```

Comment:
Nice, interesting way to define a function and reuse it a number of times. "Functional programming in Javascript" is a hot topic these days, and this is the type of problem-solving is representative of it.